components/kyma-environment-broker/internal/process/provisioning/ias_type.go

```diff
@@ -35,16 +35,21 @@ func (s *IASType) ConfigureType(operation internal.ProvisioningOperation, runtim
 		return 0, nil
 	}
 
-	spb := s.bundleBuilder.NewBundle(operation.InstanceID)
-	err := spb.FetchServiceProviderData()
-	if err != nil {
-		return s.handleError(operation, err, log, "fetching ServiceProvider data failed")
-	}
+	for spID := range ias.ServiceProviderInputs {
+		spb, err := s.bundleBuilder.NewBundle(operation.InstanceID, spID)
+		if err != nil {
+			return s.handleError(operation, err, log, "failed to create ServiceProvider Bundle")
```

Comment:
Like in `ias_registration` you can directly return `operation, 0, nil`, and log about the error.  Why not return an error? It is a step after success provisioning SKR and we don't want to block the whole operation, log with error will be informed about failure. (in future notification about this kind of failure will be reported)

Reply:
✔️